### DISCUSSION TITLE
Wonderful work

### DISCUSSION THREAD

**Discussion by SkanderBahrini** [tailwindlabs/tailwindcss#3603] [Kind words]
I am a  first year software engineer, although my lack of experience. i wanted to tell you that i love this project and express my gratitude  to you for providing such amazing work for us developers.   Thank you !

**Comment by simonswiss** [comment]
Hey, that's awesome to hear! ❤️

I hope you enjoy front end development, and Tailwind CSS brings you joy and fun on your journey 🎉
